tools/chpl-language-server/test/test_server.py

```diff
@@ -0,0 +1,611 @@
+    """
+
+    file = """
+
+           var x: int = 5;
+           var y: int = x;
+           var z: int = x + y;
+           {
+             var x = 10;
+             var y = x;
+           }
+           """
+
+    with source_file(client, file) as doc:
+        # Definitions link to themselves
+        await check_goto_decl_def(client, doc, pos((0, 4)), pos((0, 4)))
+        await check_goto_decl_def(client, doc, pos((1, 4)), pos((1, 4)))
+        await check_goto_decl_def(client, doc, pos((2, 4)), pos((2, 4)))
+        await check_goto_decl_def(client, doc, pos((4, 6)), pos((4, 6)))
+        await check_goto_decl_def(client, doc, pos((5, 6)), pos((5, 6)))
+
+        # References link to their definitions
+        await check_goto_decl_def(client, doc, pos((1, 13)), pos((0, 4)))
+        await check_goto_decl_def(client, doc, pos((2, 13)), pos((0, 4)))
+        await check_goto_decl_def(client, doc, pos((2, 17)), pos((1, 4)))
+        await check_goto_decl_def(client, doc, pos((5, 10)), pos((4, 6)))
+
+        # Other things don't go anywhere
+        # 'var' tokens
+        await check_goto_decl_def(client, doc, pos((0, 2)), None)
+        await check_goto_decl_def(client, doc, pos((1, 2)), None)
+        await check_goto_decl_def(client, doc, pos((2, 2)), None)
+        await check_goto_decl_def(client, doc, pos((4, 4)), None)
+        await check_goto_decl_def(client, doc, pos((5, 4)), None)
+        # numbers
+        await check_goto_decl_def(client, doc, pos((0, 13)), None)
+        await check_goto_decl_def(client, doc, pos((4, 10)), None)
+
+        assert len(client.diagnostics) == 0
+
+
+@pytest.mark.asyncio
+async def test_go_to_definition_use_standard(client: LanguageClient):
+    """
+    Ensure that go-to-definition works on standard module symbols
+    that are used or imported.
+    """
+
+    file = """
+           use IO;
+           use List, Map;
+           import Time;
+           """
+
+    mod_IO = standard_module("standard/IO.chpl")
+    mod_List = standard_module("standard/List.chpl")
+    mod_Map = standard_module("standard/Map.chpl")
+    mod_Time = standard_module("standard/Time.chpl")
+
+    with source_file(client, file) as doc:
+        await check_goto_decl_def(client, doc, pos((0, 4)), mod_IO)
+        await check_goto_decl_def(client, doc, pos((1, 4)), mod_List)
+        await check_goto_decl_def(client, doc, pos((1, 10)), mod_Map)
+        await check_goto_decl_def(client, doc, pos((2, 8)), mod_Time)
+
+        assert len(client.diagnostics) == 0
+
+
+@pytest.mark.asyncio
+async def test_go_to_definition_standard_rename(client: LanguageClient):
+    """
+    Ensure that go-to-definition works with using/importing symbols,
+    even if renaking is in use.
+    """
+
+    file = """
+           use IO as OI;
+           import IO.{ioMode as im};
+           use List only list;
+           """
+
+    mod_IO = standard_module("standard/IO.chpl")
+    mod_List = standard_module("standard/List.chpl")
```

Comment:
If its a standard module, why do you need `standard_module("standard/MODNAME.chpl")`. Can you make this shortcut in `standard_module("MODNAME.chpl")`, or even `standard_module("MODNAME")`